catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -0,0 +1,569 @@
+public class ExportCommand extends CqlCommands {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExportCommand.class);
+
+    private static final SimpleDateFormat ISO_8601_DATE_FORMAT;
+
+    static {
+        ISO_8601_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH-mm-ss.SSS'Z'");
+        ISO_8601_DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
+    }
+
+    private static final Supplier<String> name =
+            () -> "export-" + ISO_8601_DATE_FORMAT.format(Date.from(Instant.now())) + ".zip";
+
+    private static MetacardTransformer transformer;
+
+    private final int PAGE_SIZE = 64;
+
+    private Filter revisionFilter;
+
+    private JarSigner jarSigner = new JarSigner();
+
+    @Reference
+    private StorageProvider storageProvider;
+
+    @Option(name = "--output", description = "Output file to export Metacards into. Paths are absolute and must be in quotes. Will default to auto generated name inside of ddf.home", multiValued = false, required = false, aliases = {
+            "-o"})
+    String output = Paths.get(System.getProperty("ddf.home"), name.get())
+            .toString();
+
+    @Option(name = "--delete", required = true, aliases = {"-d",
+            "delete"}, multiValued = false, description = "Whether or not to delete metacards after export")
+    boolean delete = false;
+
+    @Option(name = "--archived", required = false, aliases = {"-a",
+            "archived"}, multiValued = false, description = "Equivalent to --cql \"\\\"metacard-tags\\\" like 'deleted'\"")
+    boolean archived = false;
+
+    @Override
+    protected Object executeWithSubject() throws Exception {
+        Filter filter = getFilter();
+        transformer = getTransformers(DEFAULT_TRANSFORMER_ID).stream()
+                .findFirst()
+                .orElseThrow(() -> new CatalogCommandRuntimeException(
+                        "Could not find transformer " + DEFAULT_TRANSFORMER_ID));
+        revisionFilter = initRevisionFilter();
+
+        final File outputFile = initOutputFile(output);
+        if (outputFile.exists()) {
+            printErrorMessage(String.format("File [%s] already exists!", outputFile.getPath()));
+            return null;
+        }
+
+        final File parentDirectory = outputFile.getParentFile();
+        if (parentDirectory == null || !parentDirectory.isDirectory()) {
+            printErrorMessage(String.format("Directory [%s] must exist.", output));
+            console.println("If the directory does indeed exist, try putting the path in quotes.");
+            return null;
+        }
+
+        String filename = FilenameUtils.getName(outputFile.getPath());
+        if (StringUtils.isBlank(filename) || !filename.endsWith(".zip")) {
+            console.println("Filename must end with '.zip' and not be blank");
+            return null;
+        }
+
+        SecurityLogger.audit("Called catalog:dump command with path : {}\nCurrent user: {}",
```

Comment:
I think it is ok to audit the list of IDs in the audit log.  In general I think we try to avoid auditing anything more than just the IDs so we would not include titles for example.

@codice/security